README.md

```diff
@@ -1,29 +1,45 @@
-# ScopeLift Foundry Template
+# Gitcoin Governor Bravo
 
-A simple, opinionated template for [Foundry](https://github.com/foundry-rs/foundry) projects.
+An upgrade to a "Bravo" compatible Governor for the GitcoinDAO, built using the OpenZeppelin implementation.
 
-## Overview
+## Development
 
-Features:
+### Foundry
 
-- `foundry.toml` configured with `default`, `lite`, and `ci` profiles.
-- `foundry.toml` configured with a `fmt` configuration (which you can of course change).
-- CI configured to verify contracts are within the size limit.
-- CI configured to run tests with the `ci` profile.
-- CI configured to run `slither`, integrated with GitHub's [code scanning](https://docs.github.com/en/code-security/code-scanning). _Note that code scanning integration [only works](https://docs.github.com/en/code-security/code-scanning/automatically-scanning-your-code-for-vulnerabilities-and-errors/setting-up-code-scanning-for-a-repository) for public repos, or private repos with GitHub Enterprise Cloud and a license for GitHub Advanced Security._
-- CI configured to run [scopelint](https://github.com/ScopeLift/scopelint), which:
-  - Verifies formatting of solidity (with `forge fmt`) and TOML (with `taplo`) files.
-  - Validates test names follow a convention of `test(Fork)?(Fuzz)?_(Revert(If_|When_){1})?\w{1,}`.
-  - Validates constants and immutables are in `ALL_CAPS`.
-  - Validates function names and visibility in forge scripts to 1 public `run` method per script.
-  - Validates internal functions in `src/` start with a leading underscore.
+This project uses [Foundry](https://github.com/foundry-rs/foundry). Follow [these instructions](https://github.com/foundry-rs/foundry#installation) to install it.
 
-## Development
 
-This project uses [Foundry](https://github.com/foundry-rs/foundry).
+#### Getting started
+
+```bash
+git clone git@github.com:gitcoinco/2022-Governor-upgrade.git
+forge install
```

Comment:
You actually don't need this, deps will be installed automatically the first time you run `forge build` or `forge test`

Reply:
Yeah fair enough, you don't need to do `forge build` before `forge test` either. Just enumerating the options.